tests/helpers/sample_step_implementers.py

```diff
@@ -123,3 +111,6 @@ def _run_step(self):
         return self.config.get_copy_of_runtime_step_config(
             self.environment,
             self.step_implementer_config_defaults())
+
+class NotSubClassOfStepImplementer():
+    pass
```

Comment:
Newline

Reply:
haha. both of you. i thought i pushed the change to fix that...

Reply:
its weird. locally it has it, but in ther emote it doesnt...